src/main/java/de/muenchen/isi/security/AuthoritiesEnum.java

```diff
@@ -27,7 +27,7 @@ public enum AuthoritiesEnum {
     ISI_BACKEND_ZURUECK_AN_ABFRAGEERSTELLUNG_ABFRAGE,
     ISI_BACKEND_IN_BEARBEITUNG_SETZTEN_ABFRAGE,
     ISI_BACKEND_ZURUECK_AN_SACHBEARBEITUNG_ABFRAGE,
-    ISI_BACKEND_SCHLIESSEN_ABFRAGE,
+    ISI_BACKEND_KEINE_BEARBEITUNG_NOETIG_ABFRAGE,
```

Comment:
Ist die neue Authority in den ISI-internal aufgenommen, die alte gelöscht und im ssodev importiert?

Reply:
Da die Änderung nicht abwärtskompatibel ist, kann der import in ssodev erst kurz vor dem Test geschehen.